pcsx2/GS/Renderers/Common/GSRenderer.cpp

```diff
@@ -384,6 +387,7 @@ void GSRenderer::VSync(int field)
 	{
 		ResetDevice();
 	}
+	m_scanmask_used = false;
```

Comment:
Would be better to reset this in GSState::Reset() so m_scanmask_used stays on for the entire game until a reset occurs.  then set it to true in GSState::GIFRegHandlerSCANMSK if 2 is set